### PR TITLE
Test Android Fix

### PR DESCRIPTION
**Pull request by SameerK-MSFT**
Fixes #
Android build is failing

**Changes proposed in this request**
Add frameworks for Android

**Testing**
NA

**Performance impact**
NA

**Documentation**
- [x] All relevant documentation is updated.

### CODE REVIEW

**Review by gladjohn — APPROVED** [review]
:shipit:

**Comment by gladjohn** [comment]
just change the title so we know what the fix is actually for

File: build/template-build-android-automation.yaml [hunk 1]
```diff
@@ -20,6 +25,13 @@ steps:
     TargetFolder: '$(build.artifactstagingdirectory)'
     flattenFolders: true
 
+- task: DotNetCoreCLI@2
+  displayName: 'Tmp dotnet workload restore for Desktop project'
```
Comment (pmaytak) [inline build/template-build-android-automation.yaml:29]:
```suggestion
  displayName: 'dotnet workload restore for Desktop project'
```

**Review by pmaytak — APPROVED** [review]

File: build/template-build-and-run-all-tests.yaml [hunk 2]
```diff
@@ -107,21 +107,20 @@ jobs: #Build and stage projects
...
-#  pool:
-#    vmImage: 'windows-2022'
-#  variables:
-#    runCodesignValidationInjection: false
-
-#  steps:
-#  - template: template-ios-appcenter-tests.yaml
-#    parameters:
-#      DataFileDirectory: ${{ parameters.DataFileDirectory }}
+# - job: 'RunAppCenterIosMsalTests'
```
Comment (trwalke) [inline build/template-build-and-run-all-tests.yaml:111]: so iOS tests are still not working?

**Review by trwalke — APPROVED** [review]

File: build/template-build-and-run-all-tests.yaml (hunk 2 above)
Comment (SameerK-MSFT) [reply build/template-build-and-run-all-tests.yaml:111]: iOS started failing again. I tried several times with no luck. If your branch has fix, that can be merged.

**Pull request merged**